app/src/main/java/org/simple/clinic/bp/entry/confirmremovebloodpressure/ConfirmRemoveBloodPressureDialogController.kt

```diff
@@ -27,13 +27,14 @@ class ConfirmRemoveBloodPressureDialogController @Inject constructor(
   }
 
   private fun markBloodPressureAsDeleted(events: Observable<UiEvent>): Observable<UiChange> {
-    val savedBloodPressureMeasurementStream = events
+    val savedBloodPressureMeasurementUuidStream = events
         .ofType<ConfirmRemoveBloodPressureDialogCreated>()
-        .flatMapSingle { bloodPressureRepository.measurement(it.bloodPressureMeasurementUuid) }
```

Comment:
FYI: This was done because we use a `withLatestFrom` on this stream later on which subscribes to this immediately which causes the database operation to execute on the main thread. This is probably the same reason why we have seen this happening in some other screens, which we'll have to investigate later.

Reply:
👍